packages/zero-pg/src/query.ts

```diff
@@ -82,15 +86,13 @@ export class ZPGQuery<
     ast: AST,
     format: Format,
   ) {
-    super(schema, tableName, ast, format);
+    super(schema, tableName, ast, format, 'permissions');
```

Comment:
note to self: new system for `api server`?

https://bugs.rocicorp.dev/issue/3791